proto/eth/v1alpha1/archive.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package ethereum.eth.v1alpha1;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "proto/eth/v1alpha1/beacon_block.proto";
+
+option go_package = "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1;eth";
+
+message ArchivedActiveSetChanges {
```

Comment:
Can you please add commentary to this proto?

Reply:
+1 , needs more context, so that first time readers can grasp it